saleor/graphql/webhook/subscription_payload.py

```diff
@@ -65,6 +65,89 @@ def get_event_payload(event):
     return event
 
 
+def generate_payload_promise_from_subscription(
```

Comment:
Cannot we merge this logic somehow with `generate_payload_from_subscription`, it seems similar?

Reply:
This logic was duplicated to handle the case when the executor returns `promise` instead of `payload` 
We could extract some similar parts to helpers but I'm afraid that decrease code readability. What do you think?

Reply:
I'm not insist, I am only worried that some adjustments would be done in one place and not in another.